src/schema/v2/me/__tests__/fixtures/userInterestFixture.ts

```diff
@@ -0,0 +1,50 @@
+export const userInterestFixture = {
```

Comment:
This feels like a giant fixture, is that really needed for the relatively simple spec?

Reply:
Hmm, my idea to use a recorded response from Gravity instead of creating one myself (like with [VCR[(https://github.com/vcr/vcr) in Rails). But you're right, the whole response isn't needed and I can probably instead pick the fields I need and drop the fixture...

Reply:
Interesting! That's not a bad idea, I'm not strongly opinionated one way or the other, it just jumped out at being a large fixture.